recipes/freyja/meta.yaml

```diff
@@ -42,7 +42,7 @@ requirements:
     - pysam
     - biopython
     - seaborn-base
-    - pyarrow
+    - pyarrow >=10.0.1
     - sphinx
     - sphinx_rtd_theme
```

Comment:
while we're at it: you don't happen to know anything about these two requirements? Surely, they would only be required to build the docs?

Reply:
No idea, i'm just trying to fix the bioconda recipe so the docker and singularity images work in biocontainers

Reply:
Ok, never mind and lets not touch it this time.